src/services/provider/geoResource.provider.js

```diff
@@ -158,7 +158,7 @@ export const loadBvvGeoResourceById = id => {
 };
 
 /**
- * Loader for URL-based ID: An URL-based ID must basically match the following pattern:
+ * Loader for external URL-based ID: An URL-based ID must basically match the following pattern:
  * `{url}||{extraParam1}||extraParam1.
```

Comment:
did you mean `...{url}||{extraParam1}||extraParam2.`?

Reply:
Fixed, thx